client/plots/violin.js

```diff
@@ -366,7 +364,7 @@ export function getDefaultViolinSettings(app, overrides = {}) {
 		rightMargin: 50,
 		lines: [],
 		unit: 'abs', // abs: absolute scale, log: log scale
-		plotThickness: 150,
+		plotThickness: undefined,
```

Comment:
I don't know if we should leave this blank on the UI. Xin can probably decide that.

Reply:
leaving blank can indicate the value is auto decided. good idea